.gitignore

```diff
@@ -95,4 +96,5 @@ app.*.map.json
 
 # Environment files
 *.env
-android/
\ No newline at end of file
+android/
+.gitignore
```

Comment:
This `.gitignore` file needs to be pushed otherwise the other contributors will be able to push bad files !!